pkgs/os-specific/bsd/netbsd/pkgs/make-rules.nix

```diff
@@ -0,0 +1,75 @@
+{
+  lib,
+  mkDerivation,
+  fetchNetBSD,
+  stdenv,
+  bsdSetupHook,
+  netbsdSetupHook,
+  rsync,
+}:
+
+mkDerivation {
+  path = "share/mk";
+  sha256 = "0w9x77cfnm6zwy40slradzi0ip9gz80x6lk7pvnlxzsr2m5ra5sy";
+  version = "9.2";
+  noCC = true;
+
+  buildInputs = [ ];
+  nativeBuildInputs = [
+    bsdSetupHook
+    netbsdSetupHook
+    rsync
```

Comment:
What's rsync used for?

Reply:
NetBSD's combining the extra paths, which is not as good as what we did for FreeBSD and OpenBSD. We should really switch it over to the same way of doing things as the others at some point.

Reply:
I'd like to do that in a separate PR.

Reply:
Oh yes, I meant that. It is definitely out of scope for this one,